docs/tutorials/tutorial-0.rst

```diff
@@ -1,175 +1,65 @@
-Tutorial 0 - Sandbox
-====================
+Tutorial 0 - Getting Started
```

Comment:
What's the motivation between turning "getting started" into tutorial 0? Not necessarily disagreeing with the change, just interested in hearing your motivation. 

My original thought was "getting started" is just getting ducks in a row; a tutorial should end with something that actually works or does something. Tutorial 0 isn't actually doing anything practical; it's just getting the sandbox running. Tutorial 1 is the first "actually do something useful" version.